woeip/apps/air_quality/admin.py

```diff
@@ -1,15 +1,10 @@
 from django.contrib import admin
 
-from woeip.apps.air_quality.models import Data, Device, Route, Sensor, Session, SessionData
+from woeip.apps.air_quality.models import Calibration, Collection, CollectionFile, Device, Pollutant, Sensor
```

Comment:
TimeGeo and PollutantValue are not registered in admin. I just want to confirm that this is a conscious decision to omit them.

Reply:
Good catch - it was intentional. Registering those models would allow editing individual datapoints via the admin UI, and that would be impractical and potentially risky for data integrity (especially since they are linked).